node/socketstream/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "todomvc-socketstream",
+  "description": "SocketStream TodoMVC app",
+  "author": {
+    "name": "Sindre Sorhus",
+    "email": "sindresorhus@gmail.com",
+    "url": "http://sindresorhus.com"
+  },
+  "private": true,
+  "engines": {
+    "node": ">=0.10.0"
+  },
+  "scripts": {
+    "start": "node app.js"
```

Comment:
Add ` postinstall` here as `bower install`. Then no need to run `bower install` separately!!